docs-content/using.html.md.erb

```diff
@@ -1,15 +1,14 @@
 ---
-title: Using YOUR-PRODUCT-NAME
+title: Using Datadog Application Monitoring for PCF
 owner: Partners
 ---
 
-This topic describes how to use YOUR-PRODUCT-NAME.
+This topic describes how to use Datadog Application Monitoring for PCF.
 
-##<a id='using'></a> Using YOUR-PRODUCT-NAME
+##<a id="view"></a> Monitor your application performance
```

Comment:
While I'm good with us mostly just linking to their docs, we don't have to provide a full tutorial here, I think we should give a little more context and explanation for what the buildpack does. 

However, I also think we need to add a tutorial here as to how to install the buildpack. The one I have on the buildpack readme will probably work pretty well.